include/vinecopulib/bicop/implementation/kernel.ipp

```diff
@@ -15,7 +16,7 @@ inline KernelBicop::KernelBicop()
     Eigen::VectorXd grid_points(m);
     for (size_t i = 0; i < m; ++i)
         grid_points(i) = -3.25 + i * (6.5 / static_cast<double>(m - 1));
-    interp_grid_ = tools_interpolation::InterpolationGrid(
+    interp_grid_ = std::make_shared<tools_interpolation::InterpolationGrid>(
```

Comment:
What is the advantage of doing that?

Reply:
interpolation_grid.hpp  doesn't need to be included. Hence it is no longer in every file that includes kernel.hpp (which is every file that includes bicop.hpp)

Reply:
That's cool :)